src/Device/Driver/LX_EOS/LXEosDevice.cpp

```diff
@@ -133,6 +133,50 @@ LXEosDevice::SendNewSettings(OperationEnvironment& env)
     return false;
 }
 
+bool
+LXEosDevice::WriteAndWaitForACK(const std::span<std::byte>& message,
+                                OperationEnvironment& env)
+{
+  port.FullFlush(
+    env, std::chrono::milliseconds(50), std::chrono::milliseconds(200));
+  port.FullWrite(message, env, std::chrono::milliseconds(1000));
+
+  try {
+    port.WaitForByte(
+      // ACK is 0x06 (NACK is 0x15)
+      std::byte{ 0x06 },
```

Comment:
There should be a central protocol definitions giving these magic values a good name. That way, you can avoid adding those comments everywhere. Look at  the LXNAV driver, it already has such a definition! And the Volkslogger driver has another one. Bad duplicate code, of course, but still better than your code.